demisto_sdk/commands/common/content/objects/pack_objects/abstract_pack_objects/json_content_object.py

```diff
@@ -108,3 +109,9 @@ def dump(self, dest_dir: Optional[Union[str, Path]] = None, change_log: Optional
             created_files.extend(self.readme.dump(dest_dir))
 
         return created_files
+
+    def is_list(self) -> bool:
```

Comment:
Let's change the name if it only applies to indicators

Reply:
It is currently used only by indicator types, but I thought it will be better to keep it general, so it can be used on other cases too if it will be needed (because what the method does isn't unique for indicator types). If it is not wanted, there is a class `IndicatorType` that inherits from `JSONContentObject` that I can move this method to.